src/main/scala/fs/package.scala

```diff
@@ -0,0 +1,37 @@
+package suffuse
+
+package object fs {
+  type DirectoryFiller   = net.fusejna.DirectoryFiller
+  type FileInfo          = net.fusejna.StructFuseFileInfo.FileInfoWrapper
+  type FileInfoWrapper   = net.fusejna.StructFuseFileInfo.FileInfoWrapper
+  type FlockCommand      = net.fusejna.FlockCommand
+  type FlockWrapper      = net.fusejna.StructFlock.FlockWrapper
+  type FuseException     = net.fusejna.FuseException
+  type FuseFilesystem    = net.fusejna.FuseFilesystem
+  type ModeInfo          = net.fusejna.types.TypeMode.ModeWrapper
+  type ModeWrapper       = net.fusejna.types.TypeMode.ModeWrapper
+  type NodeType          = net.fusejna.types.TypeMode.NodeType
+  type OpenMode          = net.fusejna.StructFuseFileInfo.FileInfoWrapper.OpenMode
+  type StatInfo          = net.fusejna.StructStat.StatWrapper
+  type StatWrapper       = net.fusejna.StructStat.StatWrapper
+  type StatvfsWrapper    = net.fusejna.StructStatvfs.StatvfsWrapper
+  type StructFuseContext = net.fusejna.StructFuseContext
+  type TimeBufferWrapper = net.fusejna.StructTimeBuffer.TimeBufferWrapper
+  type Timespec          = net.fusejna.StructTimespec.ByValue
+  type XattrFiller       = net.fusejna.XattrFiller
+  type XattrListFiller   = net.fusejna.XattrListFiller
```

Comment:
Not having to import them every time they come up, or playing "babysit the imports" during refactoring. Most of them are in the fuse-jna method signatures. From my POV anytime a name is unambiguous there is little to no downside to aliasing it throughout the package.